frontend/app/src/components/home/BigDebugMenu.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
```

Comment:
Does this need to hang around?

Reply:
I think it's quite useful for testing so I wouldn't mind leaving it. Shouldn't do any harm.